src/transformers/default-schema-transformer.test.ts

```diff
@@ -1,59 +1,115 @@
-import { AngularFormEngineSchemaTransformer } from './angular-fe-schema-transformer';
-import testForm from '__mocks__/forms/afe-forms/afe-schema-trasformer-form.json';
+import { DefaultFormSchemaTransformer } from './default-schema-transformer';
+import testForm from '__mocks__/forms/afe-forms/test-schema-transformer-form.json';
 
 const expectedTransformedSchema = {
   name: 'AFE form with aliased questions',
+  readonly: false,
```

Comment:
Pardon me if I might have missed this in previous PR's, will the transformer add this key automatically? and  is this in anyway related to the mode?

Reply:
If the property is present, the transformer will attempt to parse to a boolean. For more context, see [thread](https://openmrs.slack.com/archives/C04QZ5DDVMG/p1715788770872919?thread_ts=1715769639.667939&cid=C04QZ5DDVMG).